app/src/main/java/fr/free/nrw/commons/upload/worker/UploadWorker.kt

```diff
@@ -188,7 +189,7 @@ class UploadWorker(var appContext: Context, workerParams: WorkerParameters) :
                 .blockingGet()
             //Showing initial notification for the number of uploads being processed
 
-            Timber.e("Queued Contributions: " + queuedContributions.size)
+            Timber.tag("PRINT").e("Queued Contributions: " + queuedContributions.size)
```

Comment:
If we're planning to club the log messages together, then I would suggest using a more specific name.